test/functional/apps/visualize/_line_chart.js

```diff
@@ -142,10 +119,47 @@ define(function (require) {
           .catch(common.handleError(this));
         });
 
-        bdd.it('should show correct data', function pageHeader() {
+        bdd.it('should show correct chart order by Term', function pageHeader() {
 
           var remote = this.remote;
-          var expectedChartData = ['jpg 9,109', 'css 2,159', 'png 1,373', 'gif 918', 'php 445'];
+
+          // this test only verifies the numerical part of this data
+          // https://github.com/elastic/kibana/issues/8141
+          var expectedChartData = ['png 1,373', 'php 445', 'jpg 9,109', 'gif 918', 'css 2,159'];
+
+          common.debug('Order By = Term');
```

Comment:
this is more personal opinion, so whatever you do with this is totally at your discretion.

IMHO, I find the logging statements not necessary. The tests are quite constrained in scope, and a failure should very quickly point to the spot where things go wrong. To me, they help at dev-time, but can be removed once checked in.